### PR TITLE
Update JS_Cert.js

### PR DESCRIPTION
**Pull request by kamira**
const function

### CODE REVIEW

File: JS_Cert.js [hunk 1]
```diff
@@ -1,46 +1,61 @@
...
         
         userAns.push(ansHash);
         i++;
     }
-}
-
-function getPassword(userID) {
+  }
+  
+  const __get = (userID) => {
```
Comment (rixinsc) [inline JS_Cert.js:42]: Use a more descriptive function name? `__get` is too general.

File: JS_Cert.js [hunk 2]
```diff
@@ -1,46 +1,61 @@
...
-    return (num >> 23) ^ 2333;
-}
\ No newline at end of file
+
+const encrypt = (num) => {
+  if (typeof(num) != 'number') {
+    throw 'encrypt: The given argument is not a number';
+  }
+  return (num >> 23) ^ 2333;
+}
```
Comment (rixinsc) [inline JS_Cert.js:61]: Any specific reason making this into an arrow function?

**Review by rixinsc — CHANGES_REQUESTED** [review]

File: JS_Cert.js (hunk 1 above)
Comment (kamira) [reply JS_Cert.js:42]: 實際上你跟本拿不到 __get 的 function
你會用的是
const s = LockerSolver()
s.getPassword('')

File: JS_Cert.js (hunk 2 above)
Comment (kamira) [reply JS_Cert.js:61]: prevent ```this```

File: JS_Cert.js (hunk 1 above)
Comment (rixinsc) [reply JS_Cert.js:42]: 這我知道，只是我會希望內部function的名字可以更有意義一點w
你可以用`__getHash`之類的，單純`__get`沒什麼意思 OwO

File: JS_Cert.js (hunk 2 above)
Comment (rixinsc) [reply JS_Cert.js:61]: Can you elaborate further? The function hasn't used `this` in its scope while an arrow function will possibly decrease the code readability. I don't see any pros over here?

Comment (kamira) [reply JS_Cert.js:61]: 良好習慣要養成

Comment (rixinsc) [reply JS_Cert.js:61]: 也不是這樣子的啊…沒事用arrow function幹嘛 👀

**Pull request closed without merging**